docs/doc/30-reference/30-sql/20-query-syntax/dml-join.md

```diff
@@ -181,4 +183,101 @@ Output:
 103|Vancouver|Donut
 103|Vancouver|Coffee
 103|Vancouver|Soda
-```
\ No newline at end of file
+```
+
+## Left Join
+
+A *left join* returns all records from the left table, and the matching records from the right table. The result is NULL records from the right side, if there is no match.
+
+### Syntax
+```sql    
+SELECT select_list
+FROM table_a
+	LEFT [OUTER] JOIN table_b
+		ON join_condition_1
+```
+:::tip
+The OUTER keyword is optional.
+:::
+
+### Examples
+
+Imagine we have the following tables:
+
+Table "vip_info": This table stores the VIP client information.
+
+| Client_ID 	| Region    	|
+|-------------	|-----------	|
+| 101         	| Toronto   	|
+| 102         	| Quebec    	|
+| 103         	| Vancouver 	|
+
+Table "purchase_records": This table lists the purchase records for all the clients.
+
+| Client_ID 	| Item      	| QTY 	|
+|-------------	|-----------	|-----	|
+| 100         	| Croissant 	| 2,000   	|
+| 102         	| Donut     	| 3,000   	|
+| 103         	| Coffee    	| 6,000   	|
+| 106         	| Soda      	| 4,000   	|
+
+The following command returns the purchase records of all VIP clients, the purchase records will be NULL if the VIP client has no purchases:
+
+```sql    
+select purchase_records.Client_ID, purchase_records.Item, purchase_records.QTY from vip_info left join purchase_records on vip_info.Client_ID = purchase_records.Client_ID;
+```
+Output:
+```sql
+|NULL|NULL|NULL
```

Comment:
I guess the first NULL in the line should be 101

Reply:
Oh i see why it's null. To return the purchase records of all VIP clients in this case, it seems we'd better select vip_info.Client_ID.

Reply:
> it seems we'd better select vip_info.Client_ID.

LGTM

Reply:
Thank you.